internal/cephfs/clone.go

```diff
@@ -201,37 +208,18 @@ func createCloneFromSnapshot(ctx context.Context, parentVolOpt, volOptions *volu
 	return nil
 }
 
-// CloneStatus represents the subvolume clone status.
-type CloneStatus struct {
-	Status struct {
-		State string `json:"state"`
-	} `json:"status"`
-}
-
-func getCloneInfo(ctx context.Context, volOptions *volumeOptions, cr *util.Credentials, volID volumeID) (CloneStatus, error) {
-	clone := CloneStatus{}
-	args := []string{
-		"fs",
-		"clone",
-		"status",
-		volOptions.FsName,
-		string(volID),
-		"--group_name",
-		volOptions.SubvolumeGroup,
-		"-m", volOptions.Monitors,
-		"-c", util.CephConfigPath,
-		"-n", cephEntityClientPrefix + cr.ID,
-		"--keyfile=" + cr.KeyFile,
-		"--format=json",
+func (vo *volumeOptions) getCloneStatus(ctx context.Context, volID volumeID) (cephFSCloneState, error) {
```

Comment:
the function name is `getCloneStatus` and we are returning just state, status represents more details, not just state isn't it? can we rename the function from `getCloneStatus` to `getCloneState`?

Reply:
Done!